web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -112,6 +112,65 @@ const EncryptionSettingsForm = ({
   );
 };
 
+/**
+ * Allows to define snapshots enablement
+ * @component
+ *
+ * @param {object} props
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ * @param {onChangeFn} [props.onChange=noop] - On change callback
+ *
+ * @callback onChangeFn
+ * @param {object} settings
+ */
+const SnapshotsField = ({
+  settings,
+  onChange = noop
+}) => {
+  const rootVolume = (settings.volumes || []).find((i) => i.mountPath === "/");
+
+  const initialChecked = rootVolume !== undefined && rootVolume.fsType === "Btrfs" && rootVolume.snapshots;
```

Comment:
Not for now, but I think we should go for something similar to https://github.com/openSUSE/agama/blob/76c930c06e1ea4bb24fa11c023cd9a94e09337fd/web/src/client/storage.js#L55-L58 for having the Fstypes in a "constant".